src/components/App/index.js

```diff
@@ -27,12 +28,34 @@ export const App = ({ options }) => {
   const gameOver = new GameOver({
     element: document.querySelector('#gameovermodal'),
     config: config,
-    handleScoreSubmit: (result) => hallOfFame.saveResult(result),
+    handleScoreSubmit: (result) =>
+      hallOfFame.saveResult(result) || hallOfFame.update(),
   });
-  //gameOver.display(); //tymczasowe, jedyna metoda wywołania modala
+
+  //do wykorzystania także, gdy skończą się pytania
+  function onGameOver() {
```

Comment:
czy to dobre miejsce dla tej funkcji? wydaje się że ona też powinna zostać schowana w komponencie

Reply:
pytanie, w którym komponencie, bo ta funkcja korzysta z wielu obiektów dostępnych w appie. I ja zrozumiałam, że ma być tak, że app zarządza komunikacją między komponentami, więc w tym przypadku timer wykonuje callbacka o upłynięciu czasu, a app jakby wydaje dyspozycje innym komponentom, co mają robić.

Reply:
to też do przegadania 👍